apps/sms/index.html

```diff
@@ -50,6 +50,7 @@
     <script defer src="mocks/navigator_moz_icc_manager.js"></script>
     <!-- Firefox Desktop shims -->
     <script defer src="desktop-mock/navigator_moz_has_pending_message.js"></script>
+    <script defer src="desktop-mock/navigator_moz_set_message_handler.js"></script>
```

Comment:
why not, but maybe we could put it in moz_has_pending_message.js (and rename it to moz_system_messages.js) ?